0008-quilt-ethos.md

```diff
@@ -0,0 +1,44 @@
+# Summary
+
+Quilt has a set of Community Guidelines and Governance RFCs. These provide a set of rules, expectations and guidance on what and how certain aspects of the project should be managed. After some reflection, it seems that a set of "Values" had not been define; the aim of this document is to set these values.
+
+# Motivation
+
+This can be seen as a derivate or inspiration from the Python Projects [PEP20](https://www.python.org/dev/peps/pep-0020/). I (ToffeeMax) have found this document useful when trying to both write better code using Python and as a general set of development principles. PEP20 could partially be seen as non-serious, however it is referenced within Python frequently. 
+
+The can be refered back to as more of a "Why" a decision was made - instead of the rigid proccess of HOW that other RFCs intend.
+
+
+# Explanation
+
+The proposed to aphorisms are:
+
+1. Do not let perfect be the enemy of actually shipping an idea to usage.
+2. Modular, Simple and Flexible should be the goal of all design.
+3. Collaborate to Innovate
+4. Keep disagreements internal -- always.
+5. Consulting with many voices is good, but at some point you have to make a decision.  
+6. All players deserve a chance to explain themselves and have actions against them be reviewed by someone not involved originally.
+7. Tiny Potato got old -- time to find something new
```

Comment:
This feels almost like it's trying to have the vibe of the Zen's last line, but it's missing the substance of it